cmake/onnxruntime_python.cmake

```diff
@@ -143,7 +143,7 @@ elseif (APPLE)
     INSTALL_RPATH_USE_LINK_PATH FALSE)
 else()
   target_link_libraries(onnxruntime_pybind11_state PRIVATE ${onnxruntime_pybind11_state_libs} ${onnxruntime_EXTERNAL_LIBRARIES})
-  set_property(TARGET onnxruntime_pybind11_state APPEND_STRING PROPERTY LINK_FLAGS " -Xlinker -rpath=\$ORIGIN")
+  set_property(TARGET onnxruntime_pybind11_state APPEND_STRING PROPERTY LINK_FLAGS " -Xlinker -rpath=\\$ORIGIN")
```

Comment:
do we need to do something similar for libonnxruntime.so ? 
e.g. https://github.com/microsoft/onnxruntime/commit/76fbb54288e789a51d9eeba8fe1e44f0ca7f7b5a